### PR TITLE
Setup Travis CI and Coveralls.io

### PR DESCRIPTION
**Pull request by pbteja1998**
- Set up Travis CI
- Set up Coveralls.io

### CODE REVIEW

File: .coverage [hunk 1]
```diff
@@ -0,0 +1 @@
...9,747,389,780,903,904,905,906,395,908,909,911,401,985,408,836,1007,422,679,157,436,439,901,446,447,449,457,458,459,461,465,471,472,473,476,477,478,481,482,483,487,664,491,492,493,495,501,507,1023],"/home/teja/Documents/projects/coderplex-backend/venv/lib/python3.5/site-packages/PIL/ImtImagePlugin.py":[92,18,20,37,22,39,36,28,34],"/home/teja/Documents/projects/coderplex-backend/venv/lib/python3.5/site-packages/django/utils/ipv6.py":[130,4,149,6,7,8,217,11,12,93,260],"/home/teja/Documents/projects/coderplex-backend/venv/lib/python3.5/site-packages/PIL/PngImagePlugin.py":[512,514,515,517,277,279,34,35,36,37,39,40,42,299,44,46,49,563,54,55,56,57,58,59,60,61,62,63,64,577,66,67,68,327,72,585,843,76,845,78,81,342,92,349,94,354,611,102,362,623,624,625,626,627,628,629,630,631,632,633,634,635,124,639,128,134,65,650,653,657,403,661,153,158,423,840,179,841,184,185,390,200,459,204,206,209,121,219,293,808,246,505,383],"/home/teja/Documents/projects/coderplex-backend/venv/lib/python3.5/site-packages/django/db/models/constants.py":[3,6],"/home/teja/Documents/projects/coderplex-backend/venv/lib/python3.5/site-packages/django/forms/formsets.py":[128,1,3,4,5,6,7,8,9,10,11,12,13,14,237,16,19,20,21,22,
```
Comment (vinaypuppal) [inline .coverage]: Remove this file, it should be ignored in git. I updated `.gitignore` in #22 with recommended ignores and according to it `.coverage` file should be ignored.

**Review by vinaypuppal — DISMISSED** [review]

**Comment by rrmerugu** [comment]
@pbteja1998  will check the changes .. give me some time.

**Review by rrmerugu — DISMISSED** [review]

**Comment by pbteja1998** [comment]
@rrmerugu Coveralls is not working from the last 3 builds. I don't know why....

**Comment by pbteja1998** [comment]
disabling and re-enabling coveralls. Hope this solves the problem.

**Comment by pbteja1998** [comment]
@rrmerugu Can you approve this now ? I have disabled coveralls for now. We will enable it after this pr is merged.

**Review by rrmerugu — APPROVED** [review]

**Pull request merged**